File: desafio-02/anthonyfisicabsb/primos.py
primos = {2}

for num in range(1, 10001):
    ehPrimo = True

    for var in primos:
        if num % var == 0:
            ehPrimo = False

        if ehPrimo and num != 1:
            primos.add(num)

for var in primos:
    print(var)

File: desafio-02/anthonyfisicabsb/primos.f90
      program primos
      implicit none
      integer::i,j
      logical::eh_primo=.FALSE.
      !
      print *,"Lista de Primos entre 1-100000"
      print *, 2
      do j=3, 100000
        eh_primo=.TRUE.
        testar_quociente:do i=2, j-1
          if(modulo(j,i) .eq. 0) then
            eh_primo=.FALSE.
            exit testar_quociente
          end if
        end do testar_quociente
        !
        if(eh_primo) then
          print *,j
        end if
      end do
      end program primos    
